cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CruiseControlTest.java

```diff
@@ -790,6 +806,92 @@ public void testResources() {
         assertThat(ccContainer.getResources().getRequests(), is(requests));
     }
 
+    @ParallelTest
+    public void testCpuCapacityGeneration() {
+        Set<NodeRef> nodes = Set.of(
+                new NodeRef("foo-pool1-0", 0, "pool1", false, true),
+                new NodeRef("foo-pool1-1", 1, "pool1", false, true),
+                new NodeRef("foo-pool1-2", 2, "pool1", false, true)
+        );
+
+        Map<String, Storage> storage = new HashMap<>();
+        storage.put("pool1", new JbodStorageBuilder().withVolumes(new PersistentClaimStorageBuilder().withId(0).withSize("100Gi").build()).build());
+
+        // In this test case, when override is set for a broker, it takes precedence over resource request
+        ResourceRequirements resourceRequirements = new ResourceRequirementsBuilder()
+                .addToLimits("cpu", new Quantity(resourceLimitCpu))
+                .addToRequests("cpu", new Quantity(resourceLimitCpu))
```

Comment:
If you set resource and limit to the same value, how do you test which one is used? I think these need to be different values.

Reply:
I missed this. Now `resourceRequestsCpu` added with a value of 4.0